<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceHistoryImageItem</class>
 <widget class="QWidget" name="FaceHistoryImageItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="m_labelAddress">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>5</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>乌鲁木齐</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="m_labelDateTime">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-03-02 15:32:59</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="m_labelImage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(180,180,180)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>相似度</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_labelSimilar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>75%</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="m_progressBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>91</width>
     <height>12</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
